### DISCUSSION TITLE
will python 3.8 and 3.9 will work ?

### DISCUSSION THREAD

**Discussion by anukaal** [apache/airflow#18889] [Q&A]
can we use the python latest version 3.8 and 3.9 in setup.py

**Comment by potiuk** [comment]
yes

**Reply by anukaal** [reply]
> yes

So, we can update the setup.py with the latest version!

**Reply by potiuk** [reply]
Not sure what you are asking for , but 3.8 and 3.9 work for Airflow in general. If you explain in detail what you are planning to do, I can help more but for now I have no idea (but again latest 3.8 and 3.9 are working and tested in Airflow - we upgrade airlflow base images automatically as they are published by PSF).
